osu.Game/Graphics/Containers/LinkFlowContainer.cs

```diff
@@ -80,6 +81,25 @@ public IEnumerable<Drawable> AddLink(IEnumerable<SpriteText> text, string url, L
         public IEnumerable<Drawable> AddUserLink(User user, Action<SpriteText> creationParameters = null)
             => createLink(AddText(user.Username, creationParameters), user.Username, null, LinkAction.OpenUserProfile, user.Id.ToString(), "View profile");
 
+        public void AddMetadataLinks(string text, MetadataType type)
```

Comment:
why can't this be done at the source? seems weird to add it to LinkFlowContainer when it's only used in one place.

Reply:
Used in both Info.cs and BeatmapDetails.cs.

Reply:
Both are `MetadataSection` classes. make a base class for that instead.